tidy3d/components/base_sim/simulation.py

```diff
@@ -37,28 +37,6 @@ class AbstractSimulation(Box, ABC):
     )
     """
     Background medium of simulation, defaults to vacuum if not specified.
-
-    See Also
-    --------
-
-    `Material Library <../material_library.html>`_:
-        The material library is a dictionary containing various dispersive models from real world materials.
-
-    `Index <../mediums.html>`_:
-        Dispersive and dispersionless Mediums models.
-
-    **Notebooks:**
-
-    * `Fitting dispersive material models <../../notebooks/Fitting.html>`_
-
-    **Lectures:**
-
-    * `Modeling dispersive material in FDTD <https://www.flexcompute.com/fdtd101/Lecture-5-Modeling-dispersive-material-in-FDTD/>`_
-
-    **GUI:**
-
-    * `Mediums <https://www.flexcompute.com/tidy3d/learning-center/tidy3d-gui/Lecture-2-Mediums/>`_
-
     """
 
     structures: Tuple[Structure, ...] = pd.Field(
```

Comment:
should docstrings for structures be also cleared from FDTD related info? I.e., starting from `**Usage Caveats**`